src/interpreter/executor/interpret.js

```diff
@@ -302,7 +247,7 @@ function assigningVar(buffer, currentState, currExpression, inputLine) {
 }
 
 // If we are taking in console input
-function acceptingConsole(buffer, currExpression, inputLine) {
+function acceptingConsole(buffer, currentState, currExpression, inputLine) {
```

Comment:
I don't think this is necessary to add.
If we have a console log we will always consume it and only call this function once so we do not care about input state.

We can delete the line `currentState = stateEnum.CONSUMED` and just return consumed always in the return tuple